css/style.css

```diff
@@ -0,0 +1,235 @@
+* {
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    font-family: "Muli", sans-serif;
```

Comment:
Include a font that is web-safe ... like Arial or Verdana ... before sans-serif.